pkgs/development/python-modules/copier/default.nix

```diff
@@ -0,0 +1,62 @@
+{
+  lib,
+  git,
+  python3,
```

Comment:
in this file all of python3 should be python, that is what nixpkgs-vet is hinting at. worked locally